css/style.css

```diff
@@ -877,9 +875,11 @@ body {
     .nav .nav-item {
         font-size: 1.1rem;
     }
-    .nav-list{
-        padding:0.2rem;
+
+    .nav-list {
+        padding: 0.2rem;
     }
+
     .nav .nav-list li {
         padding: 0.5rem 0.2rem;
```

Comment:
You can use px here if the padding is too small . rem's can be used if the unit is anything above 10px

Reply:
Ok